clients/drcachesim/common/trace_entry.h

```diff
@@ -312,8 +312,8 @@ struct _offline_entry_t {
         struct {
             // This describes the entire basic block.
             uint64_t modoffs:33;
-            uint64_t modidx:12;
-            uint64_t instr_count:16;
+            uint64_t modidx:16;
+            uint64_t instr_count:12;
```

Comment:
I would suggest either a comment here advising that a change here needs to also update the hardcoded sizes in the 2 other files, or to use named constants (as is done with EXT_VALUE_A_BITS, e.g.) -- I would vote for the named constants.